appserver/distributions/pom.xml

```diff
@@ -172,6 +172,7 @@
                         <attach>false</attach>
                         <appendAssemblyId>false</appendAssemblyId>
                         <useProjectArtifact>false</useProjectArtifact>
+                        <ignoreMissingDescriptor>true</ignoreMissingDescriptor>
```

Comment:
dtto, ignoreMissingDescriptor is already set to false, which I believe is correct